mobile/src/components/Button.tsx

```diff
@@ -0,0 +1,19 @@
+import { Pressable, Text, View } from 'react-native';
+
+function ButtonPill(props) {
+  return (
+    <Pressable className= "rounded-full bg-[#2D1C70] opacity-95 h-7 items-center justify-center" >
+        <Text className='text-white margin mx-5'>{props.name}</Text>
+    </Pressable>
+  );
+}
+
+function ButtonRounded(props) {
+  return (
+    <Pressable className= "rounded-md bg-[#2D1C70] opacity-95 h-7 items-center justify-center" >
```

Comment:
For the colors I think it is good practice to use predefined colors. Currently for web we are using slate and indigo colors included in tailwind (No need to extend the theme in the config file). 

This way you can easily pick the shade too (In this case it'd be something like bg-indigo-600)